src/test/server.test.ts

```diff
@@ -0,0 +1,116 @@
+import { describe, test, expect } from "vitest";
```

Comment:
@hmrtn @vacekj is there any reason we need this extra test file? it looks like it's a duplicate of `test/http/app.test.ts` it has the exact same tests rewritten in slightly different ways

Reply:
I don't think so, it might just be a duplicate from resolving merge conflicts. Can delete in a further PR.

Reply:
Awesome I am deleting it in this PR since it was breaking my build: https://github.com/gitcoinco/allo-indexer/pull/55